bin/fastlane

```diff
@@ -91,7 +91,7 @@ class FastlaneApplication
       c.option '-f', '--force', 'Overwrite the existing README.md in the ./fastlane folder'
 
       c.action do |args, options|
-        ff = Fastlane::FastFile.new(File.join(Fastlane::FastlaneFolder.path || '.', 'Fastfile'))
+        ff = Fastlane::FastFile.new(Fastlane::FastlaneFolder.fastfile_path)
```

Comment:
:+1:  it's better to have a method with explicit action name than an instantiation.

Note that fastfile_path can return `nil` in which case `FastFile.new` does nothing. Is that path somehow expected ? Should it fail explicitly instead of silently do nothing ?